dbms/src/Storages/Page/V3/PageDirectory.cpp

```diff
@@ -1505,15 +1526,18 @@ PageDirectory<Trait>::getEntriesByBlobIds(const std::vector<BlobFileId> & blob_i
         const auto & version_entries = page_iter->second;
         // the latest entry with version.seq <= ref_id.create_ver.seq
         auto entry = version_entries->getLastEntry(ver.sequence);
-        RUNTIME_CHECK(entry.has_value(), ref_id, ori_id, ver);
```

Comment:
I think check the version_entries type and skip explicit is better. Check out the changes in here https://github.com/lidezhu/tics/pull/10

Reply:
Merged.